packages/wdio-cli/src/constants.js

```diff
@@ -114,6 +115,45 @@ export const QUESTIONNAIRE = [{
     message: 'What is the port on which that service is running?',
     default: '80',
     when: /* istanbul ignore next */ (answers) => answers.backend.indexOf('different service') > -1
+}, {
+    type: 'input',
+    name: 'exp_env_access_key',
```

Comment:
Can we use camelCase for property names? I know we haven't been consistent (e.g. `env_user`) but we shouldn't continue with this.